docs/semp.md

```diff
@@ -0,0 +1,416 @@
+| :microscope: Requirements | Drive | Spreadsheet
+| :triangular_ruler: Software Design Document (SDD) | GitHub | Tightly linked to code.<br />Markdown supports sequence diagrams.
+| :mailbox_with_mail: Interface Control Document (ICD) | GitHub | Tightly linked to code.
+| :heavy_check_mark: Verification & Validation (V&V) Document | GitHub | Code Snippets
+| :diamond_shape_with_a_dot_inside: Configuration Management (CM) Guide | GitHub | Code Snippets
+| :video_game: Human Systems Integration (HSI) Document | Google Drive | Lots of Pictures
+
+### :black_nib: Descriptions
+
+#### :thought_balloon: Concept of Operations (CONOPS)
+
+A high-level overview of a module or product.
+
+Defines the scope of development, support, and testing.
+
+Relevant:
+- [Arrow CONOPS Template](https://docs.google.com/document/d/1AhwyUmMH1rbmqMcTd1EyFIDcn-ItmaPxc7lKdZSf-q8/edit?usp=sharing)
+- [NASA CONOPS Template](https://www.nasa.gov/seh/appendix-s-concept-of-operations/)
+
+#### :running: User Stories
+
+User stories are statements of action, intent, or desire from a user of the product.
+
+User stories have the following attributes:
+- **UID**
+  - A unique integer identifier
+  - e.g. 0x123412ac
+- **Label**
+  - An easier name for this user story
+  - e.g. US-PILOT-0001
+- **Primary Text**
+  - "As a passenger, I want..."
+- **Rationale**:
+  - Why this is worded the way it is, why it exists
+
+
+UID | Label | Primary Text | Rationale
+--- | --- | --- | --- |
+0x0234abcd | US-CARGO-0001 | As a passenger, I want to... | etc.
+
+#### :microscope: Requirements
+
+Requirements are "shall" or "must" statements specific to implementation.
+
+Requirements have the following attributes:
+- **UID**
+  - A unique integer identifier
+  - e.g. 0x0ABCD123
+  - Never changes, allows Label to be changed
+- **Label**
+  - Easier name for this requirement
+  - e.g. "REQ-SCHED-0001"
+- **Primary Text**
+  - *"The module shall..."*
+- **Rationale**
+  - Why this requirement is needed
+- **Notes**
+  - Additional notes by developers or testers
+- **Parent**
+  - All requirements (except at the top level) must have a parent requirement
+  - e.g. "REQ-0001"
+- **Children**
+  - Any requirements that extend from this requirement
+- **Linked User Story**
+  - What user stories is this requirement addressing?
+
+*Example*:
+
+UID | Label | Primary Text | Rationale | Parent | Children | User Stories
+--- | --- | --- | --- | --- | --- | --- 
+0x0234abcd | REQ-0005 | The module shall... | Because... | REQ-0001 | REQ-0006<br/>REQ-0020 | US-CARGO-0001
+
+Relevant:
+- [NASA How to Write a Good
+Requirement](https://www.nasa.gov/seh/appendix-c-how-to-write-a-good-requirement/)
+
+#### :triangular_ruler: Software Design Document (SDD)
+
+Document containing:
+- State Machine Sequence Diagrams
+- Explanation of libraries
+- Explanation of global variables
+- Explanation of memory allocations
+
+Relevant:
+- [Arrow Markdown SDD Template](./templates/sdd.md)
+
```

Comment:
Broken link

Reply:
Changed to the svc-template-rust/docs/sdd.md file

Reply:
https://github.com/Arrow-air/se-services/pull/1/commits/158ed78fed2da6cd5c4e468c9f21d75e25cb421d